.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: python
+
+python: 
+- "2.7"
+- "3.6"
```

Comment:
not sure if it will work with Python 3 but since there is no tests it should pass! :|

Reply:
Will keep it for now :stuck_out_tongue: